### PR TITLE
Globus documentation initial commit

### PR DESCRIPTION
**Pull request by avapirev**
Cheers.

### CODE REVIEW

File: glossary_globus.tex [hunk 1]
```diff
@@ -0,0 +1,17 @@
+\newglossaryentry{Collection}
+{ name={Collection},
+  description={A collection is a named location containing data you can access with Globus. Collections can be hosted on many different kinds of systems, including campus storage, HPC clusters, laptops, Amazon S3 buckets, Google Drive, and scientific instruments. When you use Globus, you don’t need to know a physical location or details about storage. You only need a collection name. A collection allows authorized Globus users to browse and transfer files. Collections can also be used for sharing data with others, for data publication, and for enabling discovery by other Globus users.},
```
Comment (stdweird) [inline glossary_globus.tex]: i would renmae this to "Globus Collection" of some sort, as the explanation looks globus-only  (and fyi, irods also has the concept of a collection).

File: Makefile [hunk 2]
```diff
@@ -3,7 +3,7 @@ defines = ""
 all_os = linux mac windows
 all_site = antwerpen brussel gent leuven
 all_doc_os = intro-HPC intro-Linux
-all_doc_noos = perfexpert
+all_doc_noos = perfexpert intro-Cloud intro-Globus
```
Comment (stdweird) [inline Makefile]: this should be `intro-Data`

File: glossary_globus.tex [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+{ name={Globus Connect},
+  description={Globus Connect is used to host collections.},
+}
+\newglossaryentry{Endpoint}
+{ name={Endpoint},
+  description={An endpoint is a server that hosts collections. If you want to be able to access, share, transfer, or manage data using Globus, the first step is to create an endpoint on the system where the data is (or will be) stored. An endpoint can be a laptop, a personal desktop system, a laboratory server, a campus data storage service, a cloud service, or an HPC cluster.},
+}
+\newglossaryentry{Globus}
+{ name={Globus},
+  description={Globus is a file sharing platform.},
```
Comment (stdweird) [inline glossary_globus.tex]: at the very least, glousb is a file transfer platform.

File: glossary_globus.tex (hunk 1 above)
Comment (stdweird) [inline glossary_globus.tex]: make sure lines are no longer than 120 chars. it is very hard comment on very long lines, adn it does not matter for the formatting to split them over multiple lines

File: intro-Globus/ch_Access.tex [hunk 4]
```diff
@@ -0,0 +1,32 @@
+\chapter{Access to the VSC Cloud}
+
+The Globus platform enables developers to provide robust file transfer, sharing and search capabilities within their own research data applications and services, while leveraging advanced identity management, single sign-on, and authorization capabilities.
+
+\section{Log in with an existing identity}\label{login}
+
+Visit \href{https://www.globus.org}{www.globus.org} and click \emph{Login} at the top of the page. On the Globus login page, choose an organization you are already registered with, such as your school or your employer.
```
Comment (stdweird) [inline intro-Globus/ch_Access.tex]: or use `https://www.globus.org/app/login`?

File: intro-Globus/ch_Access.tex [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+\begin{center}
+\includegraphics[width=0.5\textwidth]{img/access-login-screen.png}
+\end{center}
+
+When you find it, click \emph{Continue}. If you cannot find your organization in the list, you can use Google, ORCID, or Globus ID, all three of which allow you to create new accounts if you do not already have one.
```
Comment (stdweird) [inline intro-Globus/ch_Access.tex]: this guide is for vsc user, if they do not find their origanoisation, they should contact support `data@vscentrum.be`

File: intro-Globus/ch_Access.tex [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+You will be redirected to your organization's login page. Use your credentials for that organization to login.
+
+Some organizations will ask for your permission to release your account information to Globus.
```
Comment (stdweird) [inline intro-Globus/ch_Access.tex]: you should mentiion that (for most organisations) this is only the first time

File: intro-Globus/ch_FileManager.tex [hunk 7]
```diff
@@ -0,0 +1,42 @@
+\chapter{Managing and transferring files}
+
+\section{The File Manager}\label{file-manager}
+
+After you have signed up and logged in to Globus, you will begin at the \emph{File Manager}. The first time you use the \emph{File Manager}, all fields will be blank.
```
Comment (stdweird) [inline intro-Globus/ch_FileManager.tex]: `sign up`?

File: intro-Globus/ch_FileManager.tex [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+
+\begin{center}
+\includegraphics[width=0.5\textwidth]{img/filemanager-1.png}
+\end{center}
+
+\section{Collections and Endpoints}\label{collections-endpoints}
+
+A \gls{Collection} is a named location containing data you can access with Globus. Collections can be hosted on many different kinds of systems, including campus storage, HPC clusters, laptops, Amazon S3 buckets, Google Drive, and scientific instruments. When you use Globus, you do not need to know a physical location or details about storage. You only need a collection name. A collection allows authorized Globus users to browse and transfer files. Collections can also be used for sharing data with others, for data publication, and for enabling discovery by other Globus users. \gls{Globus Connect} is used to host collections.
```
Comment (stdweird) [inline intro-Globus/ch_FileManager.tex]: this is copy/paste from the gloassary. i would not do that, it is very redundant.

File: intro-Globus/ch_FileManager.tex [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+
+An \gls{Endpoint} is a server that hosts collections. If you want to be able to access, share, transfer, or manage data using Globus, the first step is to create an endpoint on the system where the data is (or will be) stored.
```
Comment (stdweird) [inline intro-Globus/ch_FileManager.tex]: creationg and endpoint is advanced use for the vsc users (except when talking about their own laptop).

File: intro-Globus/ch_ShareData.tex [hunk 10]
```diff
@@ -0,0 +1,28 @@
+\chapter{Data Sharing}
+
+To share data, you will create a guest collection and grant your collaborators access as described in the instructions below. If you like, you can designate other Globus users as "access managers" for the guest collection, allowing them to grant or revoke access privileges for other Globus users.
```
Comment (stdweird) [inline intro-Globus/ch_ShareData.tex]: have you tried this? i don'tthink this already works

File: intro-Globus/title.tex [hunk 11]
```diff
@@ -0,0 +1,47 @@
...
+
+This document is a hands-on guide to the \gls{Globus} file sharing platform.  It should
+complement the official documentation at
+\url{https://docs.globus.org}.
+
+
+\strong{\underbar{Contact Information:}}
+
+
+For all questions concerning the Globus file sharing platform, please contact \globusinfo.
```
Comment (stdweird) [inline intro-Globus/title.tex]: this should ne `data@vscentrum.be`

File: macros.tex [hunk 12]
```diff
@@ -52,9 +52,13 @@
 
 \macro{\cloudinfo}{cloud@vscentrum.be}
 
+\macro{\globusinfo}{globus@vscentrum.be}
```
Comment (stdweird) [inline macros.tex]: `data@vscentrum.be`

**Comment by stdweird** [comment]
@avapirev main remaining comment is to rename everyting to "data" instead of "globus": this is the documentaion of the datacomponent, globus is only a part of it. as soon as that is done, we can merge it and published

**Comment by wdpypere** [comment]
@avapirev this PR is open for a while and changes were asked. Could you please revisit this pull request?

**Pull request closed without merging**